README.md

```diff
@@ -13,7 +13,9 @@ Live Url | - | -
 
 Here's what it does:
 
-1. 🤯 -- This need to be filled in. Sorry!
+1. Provides a minimalist and intuitive feel to planning out your schedule and day to day life.
+2. Keeps your life organized and easily displayed to stay on task.
+3. Has multiple use functionality! Use it for yourself, your family or your co-workers!
```

Comment:
"Use it for... your co-workers" sounds eery lol
3. -> `Supports a variety of shareable elements to aid organization.`